drivers/hmis/app/graphql/types/application/client_access_summary.rb

```diff
@@ -14,6 +14,8 @@ class Application::ClientAccessSummary < Types::BaseObject
     field :client_name, String, null: true
 
     def client_name
+      return client&.masked_name unless current_permission?(permission: :can_view_client_name, entity: client)
+
       client&.brief_name
```

Comment:
This `client&.brief_name` made me wonder if instead of hiding the client name completely, we could show the first letter of each name (essentially initials) if you don't have permission to see the client name.  That way we could leave it in the search results, and there would be something to pass along when discussing details of a client.